designcompose/src/main/java/com/android/designcompose/squoosh/SquooshRoot.kt

```diff
@@ -653,7 +657,10 @@ fun SquooshRoot(
                         appContext = LocalContext.current,
                         scrollOffset,
                     )
-                    .semantics { sDocRenderStatus = DocRenderStatus.Rendered }
+                    .semantics {
+                        sDocRenderStatus = DocRenderStatus.Rendered
+                        sDocRenderText = textList
```

Comment:
This sets a list of all the text that is rendered in the tree starting from the root.